src/main/java/com/metamx/metrics/JvmMonitor.java

```diff
@@ -114,4 +127,28 @@ public boolean doMonitor(ServiceEmitter emitter)
 
     return true;
   }
+
+  private void emitGcMetrics(ServiceEmitter emitter)
+  {
+    ImmutableMap<String, Long> newGcCounters = ImmutableMap.of(
+      "jvm/gc/young/cpu", youngGcCpu.getNanos(),
```

Comment:
yes, I asked it only because we should pay attention to attribute name changing. We should check that nothing important will be broken and update all dependent dashboards after release.